ts/src/profilers/profile-serializer.ts

```diff
@@ -248,6 +248,17 @@ export function serializeTimeProfile(
 export function serializeHeapProfile(
     prof: AllocationProfileNode, startTimeNanos: number,
     intervalBytes: number): perftools.profiles.IProfile {
+  // add node for external memory usage.
+  // tslint:disable-next-line: no-any
+  const memoryUsage: {external: number} = process.memoryUsage() as any;
```

Comment:
suggestion: the following might get rid of the unnecessary intermediate `memoryUsage`:

```ts
const {external} : {external: number} = process.memoryUsage() as any;
```

Reply:
Done.